sdk/eventhub/event-hubs/src/eventProcessor.ts

```diff
@@ -89,11 +103,33 @@ export class EventProcessor {
    *
    * @return {Promise<void>}
    */
-  async start(): Promise<void> {}
+  async start(): Promise<void> {
+    const partitionIds = await this._eventHubClient.getPartitionIds();
+    const partitionContext: PartitionContext = {
+      partitionId: partitionIds[0],
+      consumerGroupName: this._consumerGroupName,
+      eventHubName: this._eventHubClient.eventHubName
+    };
+    const partitionProcessor = this._partitionProcessorFactory(
+      partitionContext,
+      "checkpointManager" as any
+    );
+    this._partitionPump = new PartitionPump(
+      this._eventHubClient,
+      partitionContext,
+      partitionProcessor,
+      this._processorOptions
+    );
+    this._partitionPump.start(partitionIds[0]);
```

Comment:
I think this should be awaited so that any errors thrown by `partitionPump.start` will bubble up.

Reply:
Oopss!!